lib/peep.rb

```diff
@@ -0,0 +1,39 @@
+require 'PG'
+
+class Peep
+
+  attr_reader :id, :message, :time
+
+  def initialize(id:, message:, time:)
+    @id = id
+    @message = message
+    @time = time
+  end
+
+  def self.show
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
+    result.map do |peep|
+      Peep.new(id: peep['id'], message: peep['message'], time: peep['time'])
+    end
+  end
+
+  def self.post(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    time = Time.new.strftime('%a, %d %b %Y %H:%M:%S')
```

Comment:
PostgreSQL has the NOW() default value that sets the time in the database for you - might be worth considering so that you don't have to generate the time every time you want to create a peep